### PR TITLE
Add files via upload, could you also add this to the homepage?

### PR DESCRIPTION
**Pull request by ChicoMarxist**

### CODE REVIEW

File: intro.html [hunk 1]
```diff
@@ -5,13 +5,15 @@
         <meta charset="utf-8">
         <meta http-equiv="X-UA-Compatible" content="IE=edge">
         <title>About us</title>
+        <link rel="stylesheet" href="main.css">
     </head>
-    <body>
```
Comment (NotSoNooby) [inline intro.html:9]: WHERES THE BODY!?

File: intro.html [hunk 2]
```diff
@@ -5,13 +5,15 @@
...
     <h1>About us</h1>
+    
     <p>The Cornucopia is a project that in time, we hope to document every dish, food, and vegetable, and help those curious enough to find where they can find them.</p>
     <h1></h1>
     <p>We also wanted to have articles that got you straight to the point, Without having to read the author's life story about how swimming in a lake gave them all the skills they needed in life.</p>
     <h1></h1>
     <p>From Ducks to Watercress, Halva to Lollypops, You will find out what, and will find out where.</p>
-    </body>
```
Comment (NotSoNooby) [inline intro.html:16]: AGAIN WHERES THE BODY!?

File: intro.html [hunk 3]
```diff
@@ -5,13 +5,15 @@
...
+
+
 </html>
```
Comment (NotSoNooby) [inline intro.html:19]: ALEX WHERES THE FIRST <HTML> AT THE BEGINNING!?

EDIT: nvm found it :D

**Review by NotSoNooby — CHANGES_REQUESTED** [review]

**Comment by ChicoMarxist** [comment]
ghrhebjwrhkfgfbawhgbasaejhgyh

**Comment by NotSoNooby** [comment]
good i see bodies now. will merge

**Pull request merged**